### PR TITLE
Reformat the contributing doc to make it more readable

### PR DESCRIPTION
**Pull request by dac09**
Use collapsibles to help readability of the contributing docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ Before interacting with the Redwood community, please read and understand our [C
 - [Contributing](#contributing)
   - [Local Development](#local-development)
     - [Install Dependencies](#install-dependencies)
-    - [Copy and Watch](#copy-and-watch)
+    - [Copy and Watch](#option-a:-copy-and-watch)
       - [Specifying a RW_PATH](#specifying-a-rw_path)
-    - [Local Package Registry Emulation](#local-package-registry-emulation)
+    - [Local Package Registry Emulation](#op[tion-b:-local-package-registry-emulation)
```
Comment (Tobbe) [inline CONTRIBUTING.md:14]: stray square bracket?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ Before interacting with the Redwood community, please read and understand our [C
     - [fix-bins (alias fix)](#fix-bins-alias-fix)
     - [install (alias i)](#install-alias-i)
 
+---
```
Comment (Tobbe) [inline CONTRIBUTING.md:29]: Personally, I don't think these extra <hr>s are needed. But obviously no biggie. If more ppl want them, by all means, go ahead! 🙂

File: CONTRIBUTING.md (hunk 1 above)
Comment (dac09) [inline CONTRIBUTING.md:14]:
```suggestion
    - [Local Package Registry Emulation](#option-b:-local-package-registry-emulation)
```
I'm not sure this works on Github anyway.

**Comment by dac09** [comment]
Easier to read the md here: https://github.com/dac09/redwood/blob/improve/contributing-readability/CONTRIBUTING.md

**Comment by dac09** [comment]
Closing this, as its getting stale anyway. Not super important to be honest!

**Pull request closed without merging**